langserver/handlers/handlers_test.go

```diff
@@ -18,30 +17,38 @@ import (
 )
 
 func initializeResponse(t *testing.T, commandPrefix string) string {
-	jsonArray, err := json.Marshal(handlers.Names(commandPrefix))
-	if err != nil {
-		t.Fatal(err)
-	}
-
 	return fmt.Sprintf(`{
 		"jsonrpc": "2.0",
 		"id": 1,
 		"result": {
 			"capabilities": {
 				"textDocumentSync": {
 					"openClose": true,
-					"change": 2
+					"change": 2,
+					"save": {}
 				},
-				"hoverProvider": true,
 				"completionProvider": {},
-				"documentSymbolProvider":true,
-				"documentFormattingProvider":true,
+				"hoverProvider": true,
+				"signatureHelpProvider": {},
+				"documentSymbolProvider": true,
+				"codeLensProvider": {},
+				"documentLinkProvider": {},
+				"documentFormattingProvider": true,
+				"documentOnTypeFormattingProvider": {
+					"firstTriggerCharacter": ""
+				},
 				"executeCommandProvider": {
-					"commands": %s
+					"commands": %q
```

Comment:
This won't result in a proper array:
https://play.golang.org/p/Xjr6_HycaF9

It will be like `["one" "two"]` (missing comma). The reason this currently works is because we only have 1 command, so it is a proper array, but once it gets to 2 command it will not work

Reply:
oh, I see, good point - I will fix that! 😅

Reply:
Fixed